d/directory-ldap-api/directory-ldap-api_ubi_8.5.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash -e
+# ----------------------------------------------------------------------------
+#
+# Package	    : directory-ldap-api
+# Version           : 2.1.2
+# Source repo	    : https://github.com/apache/directory-ldap-api
+# Tested on	    : ubi 8.5
+# Language          : java
+# Travis-Check      : true
+# Script License    : Apache License, Version 2 or later
+# Maintainer	    : Pratik Tonage <Pratik.Tonage@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+# Variables
+PACKAGE_NAME="directory-ldap-api"
+PACKAGE_VERSION=${1:-"2.1.2"}
+PACKAGE_URL=https://github.com/apache/directory-ldap-api
+
+yum install -y git wget 
+
+# Setup java environment
+yum install -y java-11-openjdk java-11-openjdk-devel 
+export JAVA_HOME=/usr/lib/jvm/java-11-openjdk
+# update the path env. variable 
+export PATH=$PATH:$JAVA_HOME/bin
+
+# Install maven
+MAVEN_VERSION=${MAVEN_VERSION:-3.8.5}
+wget http://mirrors.estointernet.in/apache/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.tar.gz
+ls /usr/local
+tar -C /usr/local/ -xzf apache-maven-$MAVEN_VERSION-bin.tar.gz
+mv /usr/local/apache-maven-$MAVEN_VERSION /usr/local/maven
+ls /usr/local
+rm apache-maven-$MAVEN_VERSION-bin.tar.gz
+export M2_HOME=/usr/local/maven
+
+# update the path env. variable
+export PATH=$PATH:$M2_HOME/bin
+
+# clone the repository
+git clone $PACKAGE_URL
+cd $PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+
+# Build 
+if ! mvn install; then
```

Comment:
As you wanted to do only build, can you make this command to `mvn install -DskipTests` ?

Reply:
Updated.